service/src/main/java/bio/terra/cbas/util/BackfillOriginalWorkspaceIdService.java

```diff
@@ -39,27 +37,18 @@ public BackfillOriginalWorkspaceIdService(
     this.methodVersionDao = methodVersionDao;
   }
 
-  @WriteTransaction
   public void backfillAll() {
-    if (cbasContextConfig
-        .getWorkspaceCreatedDate()
-        .isEqual(OffsetDateTime.parse("0000-01-01T00:00:00.000000Z"))) {
-      logger.info(
-          "Skipping BackfillOriginalWorkspaceId operation. This could indicate that workspaceCreatedDate in application.yml has not been configured properly.");
-    } else {
-      try {
-        backfillMethods();
-        backfillRunSets();
-        backfillMethodVersions();
-      } catch (Exception e) {
-        // we want CBAS to fail if this operation fails, just like a liquibase migration
-        throw new RuntimeException(
-            "There was a problem while trying to backfill original workspace IDs: %s".formatted(e));
-      }
+    try {
+      backfillMethods();
+      backfillRunSets();
+      backfillMethodVersions();
+    } catch (Exception e) {
+      // we want CBAS to fail if this operation fails, just like a liquibase migration
+      throw new RuntimeException(
```

Comment:
What happens if this is thrown? This function is called by an event hook so it might be outside of the main app processing thread, so throwing an exception might just get lost

Reply:
it fails to boot! that's why this change is needed: https://github.com/DataBiosphere/cbas/pull/255/files/2bd5e96583eb94371f90405171d6fa6852ad709a..a1b8bb7631f36b0b37a5de51be1e1c66657e8847#diff-bc668a2c9f2299cef15b222055b4b4d5311646caec2e7610e540cee18ae9b948R145